.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: "Label stale PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: -1 # We don't want to address issues
+          days-before-pr-stale: 30
+          days-before-issue-close: -1 # We don't want to close issues in this action
+          days-before-pr-close: -1 # We don't want to close PR in this action
+          stale-pr-label: stale
+          stale-pr-message: |
+            This PR has been automatically marked as stale because it has not had recent activity.
+            Please add a comment if you want to keep the issue open. Thank you for your contributions!
```

Comment:
what is this message ? there nos issue opened, only a label placed, no ?

Reply:
the message appears in a comment of the PR.